src/ServerlessAutoSwagger.ts

```diff
@@ -249,6 +249,10 @@ class ServerlessAutoSwagger {
         const functions = this.serverless.service.functions;
         Object.entries(functions).map(([functionName, config]) => {
             const events = config.events || [];
+            
+            let stagePath = this.serverless.configurationInput?.provider?.stage
+            stagePath = stagePath ? `/${stagePath}` : ""
```

Comment:
I think we can do this in a nicer way with a 'basePath' defined in the 'server' settings which are global to the whole API. 
https://swagger.io/specification/#server-object